{{ cookiecutter.folder_name }}/.git_archival.txt

```diff
@@ -0,0 +1,4 @@
+node: $Format:%H$
```

Comment:
Per https://github.com/pypa/setuptools_scm/ README

Reply:
I don't quite follow this one. This is for getting our git version information into the tag tarball so it's present without the repo and without having a built package?

Reply:
When you tag on GitHub, `git archive` is used under the hood to stash away some information about the commit that generated the archive.

Trying this locally, you'll see that:

```bash
$ git archive -o ../archive.tar HEAD
$ cd ..
$ tar xfv ./archive.tar
$ cat project_name/.git_archival.txt
node: 8275d017d71e0d2cf2921a1bae7f7364cda4d8fe
node-date: 2023-01-13T09:02:54-08:00
describe-name:
ref-names: HEAD -> master
```

This results in git tag information being stored when there is no `.git` directory. setuptools-scm can read from this file:

https://github.com/pypa/setuptools_scm/blob/b2e4f628fa7ad258d10173c5818ed65a8fddb10c/src/setuptools_scm/file_finder_git.py#L107-L121

I admittedly have not tested this mechanism, but I'd trust that it works as advertised.

Reply:
I didn't put it all together and did not realize that setuptools_scm did things other than check vcs